### ISSUE TITLE
`KeyError` when trying to `get_pings_properties` `with_processes=True` for a keyed histogram with a specific key

### ISSUE THREAD

**Issue by chutten** [mozilla/python_moztelemetry#51]
```python
subset = get_pings_properties(pings, ["payload/keyedHistograms/BLOCKED_ON_PLUGIN_INSTANCE_INIT_MS/Shockwave Flash20.0.0.306"], with_processes=True)
```
This will throw a `KeyError` if `pings` contains multiple `childPayloads` because of https://github.com/mozilla/python_moztelemetry/blob/b41cee792107e84cd42be3b40485e03538345ac8/moztelemetry/spark.py#L549

```python
result[property_name + "_children"] = reduce(lambda x, y: x + y, children) if children else None
```

The `+` calls `__add__` in `histograms.py` which creates a new `Histogram` using the left-hand-side's `name` which has been set to `BLOCKED_ON_PLUGIN_INSTANCE_INIT_MS/Shockwave Flash20.0.0.306`. Unfortunately, this fails validation because there is no histogram definition by this name.

**Comment by chutten** [comment]
A workaround is to specify `additional_histograms=<fake non-keyed histogram definitions for each KEYED_HISTOGRAM/key you are requesting>`

**Comment by vitillo** [comment]
Fixed in #55 .
